assets/src/containers/ResourcesAccessed.js

```diff
@@ -178,11 +179,14 @@ function ResourcesAccessed (props) {
   useEffect(() => {
     // Fetch data once all the setting data is fetched
     if (dataControllerLoad === 2 && !(resourceTypeFilter.length === 0)) {
-      const dataURL = `/api/v1/courses/${courseId}/resource_access_within_week?week_num_start=${weekRange[0]}&week_num_end=${weekRange[1]}&grade=${resourceGradeFilter}&resource_type=${resourceTypeFilter}`
+      const dataURL = `/api/v1/courses/${courseId}/resource_access_within_week/?week_num_start=${weekRange[0]}&week_num_end=${weekRange[1]}&grade=${resourceGradeFilter}&resource_type=${resourceTypeFilter}`
```

Comment:
I noticed that when it called this URL without the / it was getting a 301 redirect. I thought that was causing a problem but probably not, just left it in there to avoid that redirect.

What you suggest looks like a better option.